### ISSUE TITLE
Dashboard markers on POI Map

### ISSUE THREAD

**Issue by ibanix** [EDSM-NET/FrontEnd#11]
Add an option to display a pilot's personal marked systems (https://www.edsm.net/en/dashboard) on the POI Map (https://www.edsm.net/en/galactic-mapping#)

Being able to see personal markers relative to POIs is very useful for route planning.

**Comment by AnthorNet** [comment]
Does having a add POI to Dashboard works for you?
It's a middle solution between performance and separation from the GMP.

As such you could have one? or a couple of POI directly to your dashboard?

**Comment by ibanix** [comment]
No, that's not particular helpful; I want to see my personal POIs relative to all the other POIs.

Also, I found a need for this feature as an Admin for the GMP. When adding a new POI, I can not see if there are other POIs near it. I would have to add it, then check, then delete it if needed. If I could see where an arbitrary system was relative to other POIs, this would be avoided.

**Comment by ibanix** [comment]
https://forums.frontier.co.uk/showthread.php/116450-Information-The-Galactic-Mapping-Project-amp-Expedition-Hub?p=5928125&viewfull=1#post5928125

This is an example of why we need this feature. A duplicate POI entry was added by GMP staff. If we were able to add custom POIs to the map, we could easily see that these two entries referenced the same object.
